test/ui/ResultListTest.ts

```diff
@@ -143,6 +143,25 @@ export function ResultListTest() {
       });
     });
 
+    it('should tell if there are more results to display after a successful query', done => {
+      Simulate.query(test.env);
```

Comment:
I am not clear on why this test succeeds. I guess it is because the number of returned results is by default greater than can be displayed? I think it should be explicitly written. Even better would be to use the "numOfResultsBeforeScroll" setting for the number.

e.g.
```
const numOfResultsBeforeScroll= settings.numOfResultsBeforeScroll;
const fakeResults = FakeResults.createFakeResults(numOfResultsPerPage + 1);
Simulate.query(test.env, {results: fakeResults});
Defer.defer(() => {
    expect(test.cmp.hasPotentiallyMoreResultsToDisplay()).toBeTruthy();
    done();
});
```